### ISSUE TITLE
loss=0.0000

### ISSUE THREAD

**Issue by b5140522137b3d4f-github** [GWxuan/TSP3D#20]
作者您好，感谢您提供的开源代码，我根据提供的checkpoint已经成功的运行了代码，并且结果也达到了预期。但是验证集的loss一直等于0.0000，这让我非常的不解，希望能够得到作者解答，谢谢！日志如下：

[09/15 00:23:56] test INFO: Full config saved to /root/autodl-tmp/TSP3D-main/data/output/logs/test/sr3d/1757867036/config.json
[09/15 00:23:56] test INFO: {'num_target': 256, 'sampling': 'kps', 'voxel_size': 0.01, 'num_encoder_layers': 3, 'num_decoder_layers': 6, 'self_position_embedding': 'loc_learned', 'self_attend': False, 'query_points_obj_topk': 4, 'use_contrastive_align': False, 'use_soft_token_loss': False, 'detect_intermediate': True, 'joint_det': True, 'batch_size': 1, 'dataset': ['sr3d'], 'test_dataset': 'sr3d', 'data_root': '/root/autodl-tmp/TSP3D-main/data/', 'use_height': False, 'use_color': True, 'use_multiview': False, 'wo_obj_name': 'None', 'butd': False, 'butd_gt': False, 'butd_cls': False, 'augment_det': False, 'num_workers': 8, 'start_epoch': 1, 'max_epoch': 400, 'optimizer': 'adamW', 'weight_decay': 0.0005, 'lr': 0.0005, 'keep_trans_lr': 0.0005, 'text_encoder_lr': 1e-05, 'box_select_lr': 0.0004, 'lr_scheduler': 'step', 'lr_decay_epochs': [150], 'lr_decay_rate': 0.1, 'clip_norm': 0.1, 'bn_momentum': 0.1, 'syncbn': False, 'warmup_epoch': -1, 'warmup_multiplier': 100, 'checkpoint_path': '/root/autodl-tmp/TSP3D-main/data/checkpoints/ckpt_sr3d.pth', 'log_dir': '/root/autodl-tmp/TSP3D-main/data/output/logs/test/sr3d/1757867036', 'print_freq': 500, 'save_freq': 6, 'val_freq': 3, 'local_rank': 0, 'ap_iou_thresholds': [0.25, 0.5], 'rng_seed': 0, 'debug': False, 'eval': True, 'eval_train': False, 'pp_checkpoint': None, 'reduce_lr': False}
[09/15 00:25:32] test INFO: length of testing dataset: 17726
[09/15 00:26:21] test INFO: Eval: [500/17726]  
[09/15 00:26:21] test INFO: loss 0.0000 	
[09/15 00:27:01] test INFO: Eval: [1000/17726]  
[09/15 00:27:01] test INFO: loss 0.0000 	
[09/15 00:27:44] test INFO: Eval: [1500/17726]  
[09/15 00:27:44] test INFO: loss 0.0000 	
[09/15 00:28:26] test INFO: Eval: [2000/17726]  
[09/15 00:28:26] test INFO: loss 0.0000 	
[09/15 00:29:05] test INFO: Eval: [2500/17726]  
[09/15 00:29:05] test INFO: loss 0.0000 	
[09/15 00:29:47] test INFO: Eval: [3000/17726]  
[09/15 00:29:47] test INFO: loss 0.0000 	
[09/15 00:30:31] test INFO: Eval: [3500/17726]  
[09/15 00:30:31] test INFO: loss 0.0000 	
[09/15 00:31:13] test INFO: Eval: [4000/17726]  
[09/15 00:31:13] test INFO: loss 0.0000 	
[09/15 00:31:53] test INFO: Eval: [4500/17726]  
[09/15 00:31:53] test INFO: loss 0.0000 	
[09/15 00:32:37] test INFO: Eval: [5000/17726]  
[09/15 00:32:37] test INFO: loss 0.0000 	
[09/15 00:33:16] test INFO: Eval: [5500/17726]  
[09/15 00:33:16] test INFO: loss 0.0000 	
[09/15 00:33:58] test INFO: Eval: [6000/17726]  
[09/15 00:33:58] test INFO: loss 0.0000 	
[09/15 00:34:38] test INFO: Eval: [6500/17726]  
[09/15 00:34:38] test INFO: loss 0.0000 	
[09/15 00:35:18] test INFO: Eval: [7000/17726]  
[09/15 00:35:18] test INFO: loss 0.0000 	
[09/15 00:35:58] test INFO: Eval: [7500/17726]  
[09/15 00:35:58] test INFO: loss 0.0000 	
[09/15 00:36:36] test INFO: Eval: [8000/17726]  
[09/15 00:36:36] test INFO: loss 0.0000 	
[09/15 00:37:18] test INFO: Eval: [8500/17726]  
[09/15 00:37:18] test INFO: loss 0.0000 	
[09/15 00:38:03] test INFO: Eval: [9000/17726]  
[09/15 00:38:03] test INFO: loss 0.0000 	
[09/15 00:38:44] test INFO: Eval: [9500/17726]  
[09/15 00:38:44] test INFO: loss 0.0000 	
[09/15 00:39:27] test INFO: Eval: [10000/17726]  
[09/15 00:39:27] test INFO: loss 0.0000 	
[09/15 00:40:06] test INFO: Eval: [10500/17726]  
[09/15 00:40:06] test INFO: loss 0.0000 	
[09/15 00:40:46] test INFO: Eval: [11000/17726]  
[09/15 00:40:46] test INFO: loss 0.0000 	
[09/15 00:41:24] test INFO: Eval: [11500/17726]  
[09/15 00:41:24] test INFO: loss 0.0000 	
[09/15 00:42:03] test INFO: Eval: [12000/17726]  
[09/15 00:42:03] test INFO: loss 0.0000 	
[09/15 00:42:44] test INFO: Eval: [12500/17726]  
[09/15 00:42:44] test INFO: loss 0.0000 	
[09/15 00:43:23] test INFO: Eval: [13000/17726]  
[09/15 00:43:23] test INFO: loss 0.0000 	
[09/15 00:44:02] test INFO: Eval: [13500/17726]  
[09/15 00:44:02] test INFO: loss 0.0000 	
[09/15 00:44:42] test INFO: Eval: [14000/17726]  
[09/15 00:44:42] test INFO: loss 0.0000 	
[09/15 00:45:21] test INFO: Eval: [14500/17726]  
[09/15 00:45:21] test INFO: loss 0.0000 	
[09/15 00:46:02] test INFO: Eval: [15000/17726]  
[09/15 00:46:02] test INFO: loss 0.0000 	
[09/15 00:46:42] test INFO: Eval: [15500/17726]  
[09/15 00:46:42] test INFO: loss 0.0000 	
[09/15 00:47:21] test INFO: Eval: [16000/17726]  
[09/15 00:47:21] test INFO: loss 0.0000 	
[09/15 00:48:02] test INFO: Eval: [16500/17726]  
[09/15 00:48:02] test INFO: loss 0.0000 	
[09/15 00:48:44] test INFO: Eval: [17000/17726]  
[09/15 00:48:44] test INFO: loss 0.0000 	
[09/15 00:49:26] test INFO: Eval: [17500/17726]  
[09/15 00:49:26] test INFO: loss 0.0000 	
[09/15 00:49:45] test INFO: Eval: [85]  
[09/15 00:49:45] test INFO: 3dcnn Acc0.25: Top-1: 0.57261
[09/15 00:49:45] test INFO: 3dcnn Acc0.50: Top-1: 0.44268
